src/userJourneyTestConstants.ts

```diff
@@ -0,0 +1,91 @@
+import {
+    IUserJourneyThresholds,   
+    IPerfumeOptions,
+    IPerfumeConfig,
+    IThresholdTier,
+    IUserJourneyConfig,
+    IUserJourneyStepsConfig, } from './types';
+
+export const USER_JOURNEY_THRESHOLDS: IUserJourneyThresholds = {
```

Comment:
@Cdub-9 if these are test constant, move this file to `__tests__`, please.

Reply:
All files in` __tests__ `need to include a test in them or else it fails. 

I could create a new folder called `testUtils` or something in `src` and put it there if you think that would be better than having it in directly in src

Reply:
Wait what? what about https://github.com/Zizzamia/perfume.js/blob/master/__tests__/_mock.ts ?

Those are not test, but more utilities.

Reply:
I see, needed to add `/* istanbul ignore file */` as the top